utils/utils.jsx

```diff
@@ -1678,6 +1678,27 @@ export function copyToClipboard(e, data) {
     document.body.removeChild(textArea);
 }
 
+export function copyTextToClipboard(e, data) {
```

Comment:
`copyToClipboard` is only currently used by the `CopyUrlContextMenu` component, so you could remove the nearly-duplicate function and remove the extra `e` parameter as well